### ISSUE TITLE
Non-OP Player Permissions?

### ISSUE THREAD

**Issue by loooukas** [ThunderGemios10/Survival-Games#91]
Hi!
I've got your plugin working on my server, but I can't seem to get it to work with non-opped players. I have given the default class all the (below) required permissions using LuckPerms, but still no luck for clickable signs or just typing in the command, Stat board signs & commands work, but just not these commands: 

- sg.arena.join.# - Allows the user to join a certain arena
- sg.arena.join.* - Allows the user to join all arenas
- sg.arena.vote - Allows the user to vote to start (/sg vote)
- sg.arena.spectate - Allows a user to spectate a game
- sg.lobby.join - Allows the use of /sg join to join the lobby

_TLDR: whenever players try to use the join command or click the interactable signs it doesn't work unless theyre opped._

I'm running Spigot 1.8.8 on my server with LuckPerms, Essentials, and WorldEdit.

It would be really great to fix, as it is a server for about 20 friends of mine and because of the quarantine, we're meeting up online to play on the server so if you do have a fix for this, a response on github here or my Discord is: Cookies#9696 if that works better. Thanks!

(This probably isn't a bug, i'm probably just stupid)

**Comment by Remyk1** [comment]
The plugin is made for 1.12.2 and higher. 1.8.x is not supported.
